snuba/api.py

```diff
@@ -353,17 +355,16 @@ def ensure_table_exists():
             )
         )
 
-    ensure_table_exists()
-
-    if settings.CLICKHOUSE_TABLE == 'dev':
-        from snuba import migrate
-        migrate.run(clickhouse_rw, settings.CLICKHOUSE_TABLE)
+        if settings.CLICKHOUSE_TABLE == 'dev':
```

Comment:
I guess this also means that every single query is also going to run this migrate, right?

Reply:
Yeah I was just thinking that, I'll set a flag to skip if it's been initialized I guess? Unless you're planning to blow away CH tables from under Snuba?

Reply:
I think it'd be worth having a global flag so it just does it once? Or rather, if it succeeds successfully once.